src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1697,9 +1697,9 @@ let mina_commands logger =
   let group =
     List.map
       ~f:(fun (module T) -> (T.name, T.command))
-      ( [ (module Coda_shared_state_test)
-        ; (module Coda_transitive_peers_test)
-        ; (module Coda_change_snark_worker_test)
+      ( [ (* (module Coda_shared_state_test)
```

Comment:
would be good to include in the comment the status of these tests, why they're being disabled

presumably, you're leaving the comment for the possibility of reviving these tests

Reply:
these will all go away soon